src/app/pages/receive/components/receive-tokens.tsx

```diff
@@ -47,6 +83,66 @@ export function ReceiveTokens({
         onClickQrCode={onClickQrStx}
         title="Stacks"
       />
+      <ReceiveItem
+        address={btcAddressTaproot}
+        icon={<Brc20AvatarIcon />}
+        dataTestId={HomePageSelectors.ReceiveBtcTaprootQrCodeBtn}
+        onCopyAddress={async () => {
+          await copyToClipboard(btcAddressTaproot);
+          toast.success('Copied to clipboard!');
+        }}
+        // onClickQrCode={onClickQrOrdinal}
```

Comment:
I disabled `onClickQrCode` for these as it will take more work with routes which will take some more time.